utils/node_local_test.py

```diff
@@ -2707,7 +2731,10 @@ def main():
     setup_log_test(conf)
 
     server = DaosServer(conf, test_class='first')
-    server.start()
+    clean=True
+    if args.no_root:
+        clean=False
+    server.start(clean=clean)
```

Comment:
This seems to be the crux of the fix although I know not why.  Without it then the server is restarting the engine a number of times.